### PR TITLE
docs: (IAC-1205) Update SAS doc links

### PR DESCRIPTION
**Pull request by riragh**
## Changes:
Updated the SAS doc links to the latest recommended format by SAS documentation team.

### CODE REVIEW

**Review by thpang — APPROVED** [review]
LGTM

**Review by jarpat — APPROVED** [review]

**Review by dhoucgitter — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ This project helps you to automate the cluster-provisioning phase of SAS Viya pl
 
 [<img src="./docs/images/viya4-iac-azure-diag.png" alt="Architecture Diagram" width="750"/>](./docs/images/viya4-iac-azure-diag.png?raw=true)
 
-This project addresses the first of three steps in [Steps for Getting Started](https://go.documentation.sas.com/doc/en/itopscdc/default/itopscon/n12fgslcw9swbsn10rva4bp0mr2w.htm) in _SAS&reg; Viya&reg; Platform Operations_:
+This project addresses the first of three steps in [Steps for Getting Started]https://documentation.sas.com/?cdcId=itopscdc&cdcVersion=default&docsetId=itopscon&docsetTarget=n12fgslcw9swbsn10rva4bp0mr2w.htm) in _SAS&reg; Viya&reg; Platform Operations_:
```
Comment (sayeun) [inline README.md:18]: This link is missing an open parenthesis that causes the link is not displaying correctly.

**Review by sayeun — CHANGES_REQUESTED** [review]

File: docs/CONFIG-VARS.md [hunk 2]
```diff
@@ -358,12 +358,12 @@ Each server element, like `foo = {}`, can contain none, some, or all of the para
...
+Multiple SAS offerings require a second PostgreSQL instance referred to as SAS Common Data Store, or CDS PostgreSQL. For more information, see [Common Customizations](https://documentation.sas.com/?cdcId=itopscdc&cdcVersion=v_046&docsetId=dplyml0phy0dkr&docsetTarget=n08u2yg8tdkb4jn18u8zsi6yfv3d.htm#p0wkxxi9s38zbzn19ukjjaxsc0kl). A list of SAS offerings that require CDS PostgreSQL is provided in [SAS Common Data Store Requirements](https://documentation.sas.com/?cdcId=itopscdc&cdcVersion=v_046&docsetId=itopssr&docsetTarget=p05lfgkwib3zxbn1t6nyihexp12n.htm#n03wzanutmc6gon1val5fykas9aa). To create and configure an external CDS PostgreSQL instance in addition to the external platform PostgreSQL instance named `default`, specify `cds-postgres` as a second PostgreSQL instance, as shown in the example below.
```
Comment (sayeun) [inline docs/CONFIG-VARS.md:366]: Is there a reason we need to pin it to v_046?

File: README.md (hunk 1 above)
Comment (riragh) [reply README.md]: Fixed the issue.

File: docs/CONFIG-VARS.md (hunk 2 above)
Comment (riragh) [reply docs/CONFIG-VARS.md]: Good catch. This was missed in the previous change, updated the links.

**Review by sayeun — APPROVED** [review]

**Pull request merged**